models/articleModel.js

```diff
@@ -13,39 +13,33 @@ const selectArticleById = (article_id) => {
     })
 };
 
-const selectArticles = (sort_by = 'created_at', order = `DESC`) => {
-    const validSortBy = ["author",
-        "title",
-        "article_id",
-        "body",
-        "topic",
-        "created_at",
-        "votes",
-        "article_img_url"]
-    
+const selectArticles = (sort_by = 'created_at', order_by = `DESC`, topic) => {
+    const validSortBy = ["author", "title", "article_id", "body", "topic", "created_at", "votes", "article_img_url"]
     const validOrder = ["ASC", "DESC"]
+    const validTopics = ["mitch", "cats"]
```

Comment:
Are mitch and cats topics in the test data?? They won't be. This is a very brittle way to handle this. If this were to stay your filter by topics in front end just won't work. How could you find the valid topics dynamically?